inst/include/interface/rcpp/rcpp_objects/rcpp_models.hpp

```diff
@@ -749,6 +737,19 @@ class CatchAtAgeInterface : public FisheryModelInterfaceBase {
    * @brief Method to convert the model to a JSON string.
    */
   virtual std::string to_json() {
+    std::shared_ptr<fims_info::Information<double>> info =
+        fims_info::Information<double>::GetInstance();
+
+
+    std::shared_ptr<fims_model::Model<double>> model_internal =
+        fims_model::Model<double>::GetInstance();
+        
+    std::shared_ptr<fims_popdy::CatchAtAge<double>> model =
+        std::dynamic_pointer_cast<fims_popdy::CatchAtAge<double>>(
+            info->models_map[this->get_id()]);
+
+            model->do_reporting = true;
```

Comment:
will this affect SE calculations when calling sdreport? Seems like model->do_reporting would be set to false when running sdreport.